### PR TITLE
[IMP] website: correct filter names

### PR DESCRIPTION
**Pull request by sswapnesh**
Usability Improvement

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/website_views.xml [hunk 1]
```diff
@@ -241,9 +241,9 @@
                     <field name="url"/>
                     <field name="website_id" groups="website.group_multi_website"/>
                     <group string="Group By">
-                        <filter string="name" name="group_by_name" domain="[]" context="{'group_by':'name'}"/>
-                        <filter string="url" name="group_by_url" domain="[]" context="{'group_by':'url'}"/>
-                        <filter string="website"  name="my_websites" domain="[]" context="{'group_by':'website_id'}"/>
+                        <filter string="View Name" name="group_by_name" domain="[]" context="{'group_by':'name'}"/>
```
Comment (JKE-be) [inline addons/website/views/website_views.xml:244]: thx...

But why 'View Name' ?

Comment (sswapnesh) [reply addons/website/views/website_views.xml]: My Bad, Updated.

Comment (JKE-be) [reply addons/website/views/website_views.xml]: Sorry but I will prefer name ;)
You are in the tree view of menu, so what you do is group by Name really ;)

Comment (sswapnesh) [reply addons/website/views/website_views.xml]: Indeed, We should remove name from Group by. 
I kept it as It was already there in previous versions.


What do you think?

Comment (sswapnesh) [reply addons/website/views/website_views.xml]: @JKE-be  Any feedback/Improvement? (or Better if this can be handled in any BacktoBasic task)

**Comment by sswapnesh** [comment]
Done

**Comment by JKE-be** [comment]
@robodoo r+

**Comment by JKE-be** [comment]
thanks

**Pull request closed without merging**